server/src/main/kotlin/net/starlegacy/feature/starship/active/ActiveStarship.kt

```diff
@@ -87,7 +87,7 @@ abstract class ActiveStarship(
 			_world = value.world
 		}
 
-	@Deprecated("Prefer Minecraft - `net.minecraft.server.level.ServerLevel`")
+	// @Deprecated("Prefer Minecraft - `net.minecraft.server.level.ServerLevel`") fuck you
```

Comment:
no

Reply:
Na don't worry, Rattlyys pr does this
Forgot to close this one